websites/F/feds/presence.ts

```diff
@@ -0,0 +1,33 @@
+const presence = new Presence({
+		clientId: "1052344047911587920",
+	}),
+	reservedPaths: Record<string, string> = {
+		"/": "Viewing the home page",
+		"/premium": "Viewing the premium page",
+		"/login": "Thinking about logging in",
+		"/join": "Thinking about joining",
+		"/tos": "Viewing the terms of service",
+		"/policy": "Viewing the privacy policy",
+		"/dash": "Viewing the dashboard",
+		"/welcome_back": "",
```

Comment:
Put in a message or just remove the entry.

Reply:
Originally this was used as a route in between login and dashboard for some reason and it displayed an ugly message. Ill remove since it is supposed to be removed itself in the next version and this will help stop a later necessary commit.